resources/workloads.yaml

```diff
@@ -97,7 +97,7 @@ spec:
         - name: direct-csi
           image: $(REPOSITORY_ORG)/$(REPOSITORY_IMAGE):$(VERSION)
           args:
-            - "--identity=direct.csi.min.io"
```

Comment:
Is this because plugin and kustomize yaml have diverged?

Reply:
yes